.github/workflows/production-heartbeat.yml

```diff
@@ -4,9 +4,9 @@ on:
   schedule:
     # Cron syntax is "minute[0-59] hour[0-23] date[1-31] month[1-12] day[0-6]". '*' is 'any value', and multiple values
     # can be specified with comma-separated lists. All times are UTC.
-    # So this expression means "run at 45 minutes past 1, 5, and 9 AM/PM UTC". The hours were chosen so employees on
-    # US Central Time will start their day with results that are <2 hrs old.
-    - cron: '45 1,5,9,13,17,21 * * *'
+    # So this expression means "run at 45 minutes past 1, 5, and 9 AM/PM UTC". The hours were chosen so that
+    # the jobs run only close to business hours of Central Time.
+    - cron: '45 13,15,17,19,21,23 * * *'
```

Comment:
Jobs will be triggered from 7:45 AM to 5:45 PM Central time in two-hour intervals.